scrapers/Puffynetwork.yml

```diff
@@ -0,0 +1,39 @@
+name: puffynetwork
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - puffynetwork.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //dl/dd/a
+    scene:
+      Title: 
+        selector: //h2[@class="title"]/span/text()
+        replace:
+          - regex: ^(.*)[—]\s+
+            with:
+      Date:
+        selector: //dl/dt[2]/span/text()

```

Comment:
imo its better to use something like `selector: //dl/dt[text()="Released on:"]/span/text()` instead of the position `dt[2]`
although it seems to work ok as it is